astropy/io/ascii/rst.py

```diff
@@ -0,0 +1,46 @@
+# Licensed under a 3-clause BSD style license
+"""
+:Author: Simon Gibbons (simongibbons@gmail.com)
+"""
+
+from __future__ import absolute_import, division, print_function
+
+from .core import DefaultSplitter
+from .fixedwidth import (FixedWidth,
+                         FixedWidthData,
+                         FixedWidthHeader,
+                         FixedWidthTwoLineDataSplitter)
+
+class SimpleRSTHeader(FixedWidthHeader):
+    position_line = 0
+    start_line = 1
+    splitter_class = DefaultSplitter
+    position_char = '='
+
+    def get_fixedwidth_params(self, line):
+        vals, starts, ends = super(SimpleRSTHeader, self).get_fixedwidth_params(line)
+        # The right hand column can be unbounded
+        ends[-1] = None
+        return vals, starts, ends
+
+class SimpleRSTData(FixedWidthData):
+    start_line = 3
+    end_line = -1
+    splitter_class = FixedWidthTwoLineDataSplitter
+
+class SimpleRST(FixedWidth):
+    """
+    Read or write a simple reStructuredText table
```

Comment:
This docstring should be more detailed, including a link to http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#simple-tables for a description of what is meant.  It should have an example table as well as a description of what is not supported (e.g. a secondary line of `------ ----` in order to indicate column spans).